yarn.config.cjs

```diff
@@ -8,6 +8,7 @@ const { defineConfig } = require(`@yarnpkg/types`);
 
 const enforcedDevDependencies = {
   prettier: { commands: ["prettier"], ident: "prettier" },
```

Comment:
Do we want to keep this and other retained references to `prettier` in this `yarn.config.cjs` file, or are they still required?

Reply:
these are still here for blocks, I tried to keep those as untouched as possible, we can remove it though!

Reply:
Okay, this makes sense - let's keep as is. Thank you!